### PR TITLE
Rename FreeRTOS-IoT-Libraries directory

### PR DESCRIPTION
**Pull request by aggarw13**
Add `LTS-Beta1` suffix to the existing `FreeRTOS-IoT-Libraries` directories to support differentiating them from C-SDK development branch libraries that will be added later

Description
-----------
Rename the following 2 directories as:
* `FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries` -> `FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta1`
* `FreeRTOS-Plus/Demo/FreeRTOS_IoT_Libraries` -> `FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1`

Test Steps
-----------
Using custom jobs to verify that demos are building

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta1/abstractions/platform/freertos/private/taskpool/taskpool_freertos.h.unc-backup~ [hunk 1]
```diff
@@ -0,0 +1,321 @@
+/*
```
Comment (muneebahmed10) [inline FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta1/abstractions/platform/freertos/private/taskpool/taskpool_freertos.h.unc-backup~:1]: These look like uncrustify backup files. Why do we need them?

Comment (aggarw13) [reply FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta1/abstractions/platform/freertos/private/taskpool/taskpool_freertos.h.unc-backup~]: Good catch. Committed them accidentally. Removed them now

**Comment by aggarw13** [comment]
Build failing due to an issue in the OTA demo file (`aws_iot_ota_update_demo.c`) which is unrelated to the changes in this PR.
Fix will be created in a separate PR.

**Pull request merged**